File: CPP/Algorithm/Is_Vowel_or_Consonant.cpp
// Complexity: O(log(n))
// To check whether a character is vowel:

bool vow(char c) {
    set<char> st = {'A', 'E', 'I', 'O', 'U', 'a', 'e', 'i', 'o', 'u'};
    return (st.find(c) != st.end());
}

// To check whether a character is consonant:

bool con(char c) {
    set<char> st = {'A', 'E', 'I', 'O', 'U', 'a', 'e', 'i', 'o', 'u'};
    return (st.find(c) == st.end());
}

File: Python/Algorithm/Is_Vowel_or_Consonant.py
# Complexity: O(1)

# To check whether a character is vowel:

def vow(c):
    vowels = "aeiouAEIOU"
    return c in vowels

# To check whether a character is consonant:

def con(c):
    vowels = "aeiouAEIOU"
    return c not in vowels and c.isalpha()